ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```diff
@@ -87,12 +92,12 @@ tests:
   - mount_path: /creds-github
     name: ossm-github-simple-job
   timeout: 20m0s
-- as: e2e-ocp-4-15
+- as: e2e-ocp-4-16
   steps:
     cluster_profile: ossm-aws
     env:
       BASE_DOMAIN: servicemesh.devcluster.openshift.com
-      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.15.6-multi
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.16.3-multi
```

Comment:
How does this work? Do we have to always change this to get newer OCP version?

Reply:
We are the only ones using this variable: HYPERSHIFT_HC_RELEASE_IMAGE. It seems to be that is the way to update but is manual work. On ipi cluster we can set the version and the channel